Index: text2skin-1.3.1/bitmap.c
===================================================================
--- text2skin-1.3.1/bitmap.c
+++ text2skin-1.3.1/bitmap.c	2010-08-14 23:10:44.369603897 +0300
@@ -187,7 +187,7 @@
 				return false;
 			}
 			*/
-			bmp = new cBitmap(w, h, std::min((*it).depth(), 8U));
+			bmp = new cBitmap(w, h, std::min(size_t((*it).depth()), size_t(8)));
 			//Dprintf("this image has %d colors\n", (*it).totalColors());
 
 			const PixelPacket *pix = (*it).getConstPixels(0, 0, w, h);
